libcxx/include/syncstream

```diff
@@ -253,7 +255,10 @@ public:
 
   // [syncstream.syncbuf.cons], construction and destruction
 
-  _LIBCPP_HIDE_FROM_ABI explicit basic_syncbuf(streambuf_type* __obuf = nullptr)
+  _LIBCPP_HIDE_FROM_ABI basic_syncbuf()
+      : basic_syncbuf(nullptr) {}
```

Comment:
I think it's possible to test this. Before applying the LWG issue resolution, the following code would not have compiled:

```
basic_syncbuf f() {
  return {};
}
```

But after the LWG issue resolution, this compiles because the default constructor is not explicit. So I think a test like this should work:

```
basic_syncbuf buf = {};
```

Reply:
I like the `return {}` test better, so went that direction for the test.